src/orderbook/orderbooks.py

```diff
@@ -33,53 +34,33 @@ class SimpleOrderBook:
     def __init__(self, bid: Stack, ask: Stack, kafka_client: KafkaClient):
         self.bid = bid
         self.ask = ask
+        self.lock = threading.Lock()
```

Comment:
How does it work ? I would have added that in the orderbook class instead of the stack